src/lib/logic/utils/errors.ts

```diff
@@ -49,19 +62,17 @@ export function setErrors<T extends Errors = Errors>({ error, errors, handle }:
   });
 }
 
-export function setError<TKey, TErr extends Errors = Errors>({ error, errors, key }: ConfigError<TKey, TErr>) {
+export function setError<TKey extends string | number | symbol, TErr extends Errors = Errors>({ error, errors, key }: ConfigError<TKey, TErr>) {
```

Comment:
🚫 **[eslint]** <[max-len](https://eslint.org/docs/latest/rules/max-len)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 144. Maximum allowed is 130.